experimental/add_to_app/flutter_module/.metadata

```diff
@@ -4,7 +4,7 @@
 # This file should be version controlled and should not be manually edited.
 
 version:
-  revision: 2d2a1ffec95cc70a3218872a2cd3f8de4933c42f
-  channel: stable
+  revision: 532a8fed41a4f6595965f02f3edf9666ba5ebf44
+  channel: master
```

Comment:
Curious, will we change this to stable whenever the embedding goes stable? Or do we expect to keep this as master into the future?

Reply:
Change to stable once that's possible. The `experimental` directory in the repo is (in my mind) meant for samples that are currently building against master but will eventually "graduate" once the APIs they use are released to the stable channel.